delfin/tests/e2e/UpdateAccessInfo.robot

```diff
@@ -0,0 +1,119 @@
+*** Settings ***
+Documentation    Tests to verify that GET of resources
+
+Library                 RequestsLibrary
+Library                 Collections
+Library                 JSONLibrary
+Library                 OperatingSystem
+
+Suite Setup             Open Application
+Suite Teardown          Close Application
+
+*** Variables ***
+${delfin_url}           http://localhost:8190/v1
```

Comment:
Will it be better if this is moved to common place as it is used by all. If we want to run tests on remote delfin system, will be easy to run

Reply:
Tests may be invoked from individual robot files (Test suite) or to-gether with other tests. So it needs to be in the same file.